src/languages/mariadb/mariadb.keywords.ts

```diff
@@ -621,19 +671,24 @@ export const keywords = flatKeywordList({
     'WARNINGS',
     'WEEK',
     'WEIGHT_STRING',
+    'WHEN',
+    'WHERE',
     'WHILE',
     'WINDOW',
+    'WITH',
     'WITHIN',
     'WITHOUT',
     'WORK',
     'WRAPPER',
     'WRITE',
     'X509',
+    'XOR',
     'XA',
     'XML',
     'YEAR',
     'YEAR_MONTH',
     'ZEROFILL',
   ],
   constraints: ['ON DELETE', 'ON UPDATE'],
+  charset: ['CHARACTER SET'],
```

Comment:
Moved `CHARACTER SET` out of the main keywords list. Now the main lists only contain single-word keywords.

Planning to move these multi-word ones out of the `*.keywords.ts` files completely, to treat them similarly to lists like `reservedCommands`.